server/fileupdater.mjs

```diff
@@ -52,6 +60,10 @@ function updateRoutine(routine, v) {
   v<3 && v3RemoveComputeAndRandomAndApplyVariables(routine);
 }
 
+function updateProperty(properties, property, v){
```

Comment:
Why is this function being defined twice/

Reply:
That was a mistake.  I removed it in the next commit. 

![image](https://user-images.githubusercontent.com/78324099/135771369-ab1d07f2-1200-468b-8ac3-11ca3500655c.png)